proposals/cloudevents.md

```diff
@@ -0,0 +1,154 @@
+# CloudEvents
+
+**Name of project**: CloudEvents
+
+**Description**:
+
+Last year the CNCF TOC created the Serverless Working Group to investigate
+the Serverless landscape. The outputs of the WG included:
+- a [whitepaper](https://github.com/cncf/wg-serverless#whitepaper) that:
+  - defines Serverless and its terminology
+  - describes common use cases for the technology
+  - compares it with other Cloud Native technologies and **aaS environments
+  - describes the common architecture of Serverless platforms
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more serverless technology vendors and open source developers
+    to join the CNCF
+  - foster an open ecosystem by establishing interoperable APIs, in particular
+    around: Events, Deployments and Workflows
+  - provide additional education as needed
+
+One of the recommendations, interoperability around Events, was agreed to
+by the TOC and the WG began to develop a new specification for how
+Events that are transferred between an event producer and an event consumer
+should be formalized. The purpose of this would be to better enable
+interoperability between these components such that basic processing of
+the events (such as routing) can be achieved without having to require
+knowledge of the event's structure in advance, or without understanding
+the application specific data of the event.
+
+The work on this specification is currently being done within the
+CNCF's Serverless Working Group, but with the release of our first
+milestone release (v0.1), it would make sense for this work to be more
+formalized as a new sandbox project under the CNCF.
+
+The goals/roadmap of the project include:
+- moving the specification to v1.0. A baseline format for an Event
+  to enable broad adoption within the Cloud community, and in particular
+  for Serverless/FaaS implementations
+- define protocol mappings for popular transports, such as HTTP
+- define serialization mappings for popular formats, such as JSON
+
+**Statement on alignment with CNCF mission**:
+
+Being born out of the CNCF's Serverless Working Group, the CloudEvents
+project (and its members) share the CNCF's goals of promoting Cloud Native
+technologies, and offering choice to our consumers though open interoperability
```

Comment:
...th**r**ough **an** open... instead of ...though open...

Reply:
fixed